go/flags/endtoend/vtctld.txt

```diff
@@ -217,7 +217,7 @@ Usage of vtctld:
       --tablet_manager_grpc_crl string                                   the server crl to use to validate server certificates when connecting
       --tablet_manager_grpc_key string                                   the key to use to connect
       --tablet_manager_grpc_server_name string                           the server name to use to validate server certificate
-      --tablet_manager_protocol string                                   the protocol to use to talk to vttablet (default "grpc")
+      --tablet_manager_protocol string                                   Protocol to use to talk to vttablets. (default "grpc")
```

Comment:
Taking suggestions on better help text to distinguish between this and `--tablet_protocol` (this was kinda already a problem, I'm just making it even more obvious)

Reply:
> tablet_manager_protocol

AFAIK, VTGate does not use `tablet_manager_protocol`